.github/workflows/build-vault.yml

```diff
@@ -0,0 +1,116 @@
+---
+name: build_vault
+
+# This workflow is intended to be called by the build workflow for each Vault
+# binary that needs to be built and packaged. The crt make targets that are
+# utilized automatically determine build metadata and handle building and
+# packing vault. If you wish to override the default behavior of those make
```

Comment:
The latter part of this statement is no longer true as we've made inputs for all the things.